test-e2e/conditional.py

```diff
@@ -7,9 +7,10 @@
 from wcmatch import glob
 
 
-# E2E_SAFE_DEFAULT are files that typically do not affect e2e tests
+# E2E_SAFE_DEFAULT includes files that typically do not affect e2e tests.
+# `*.txt` not included here because `requirements.txt` can affect tests.
 E2E_SAFE_DEFAULT = [
-    '**/*.{md,txt}', '.github/**', 'config/**', 'docs/**', 'flake8_dcos_lint/**',
```

Comment:
Could we not add override just for `requirements.txt`? (probably failed flake8 line length ;()
```suggestion
    '**/*.{md,txt}', '!**/requirements.txt', '.github/**', 'config/**', 'docs/**', 'flake8_dcos_lint/**',
```

Reply:
The problem is that that makes any `requirements.txt` unsafe, including one that is in an otherwise ignored package.

Having looked at the code, there's no top-level `txt` files and most others will be in directories that are ignored.

The exception is `./packages/etcd/extra/README.txt`. Changing that file will run all the Calico and etcd tests, but that would be the case anyway, since a negative `!packages/etcd/**` will override any positive `**/*.txt`.

Reply:
> The problem is that that makes any `requirements.txt` unsafe, including one that is in an otherwise ignored package.

ahhh, I see, yea ;(.